test/Sentry.AspNetCore.Tests/Sentry.AspNetCore.Tests.csproj

```diff
@@ -13,6 +13,11 @@
     <ProjectReference Include="../../src/Sentry.AspNetCore/Sentry.AspNetCore.csproj" />
     <ProjectReference Include="../Sentry.Testing/Sentry.Testing.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="6.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="5.0.0"/>
```

Comment:
Should this be 6.0.0 too?
```suggestion
    <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="6.0.0"/>
```

Reply:
@bruno-garcia there is no V6
<img width="550" alt="image" src="https://user-images.githubusercontent.com/122666/174468059-061f4347-a574-4041-9a55-f674b5e525b9.png">